frontend/routes/modelSettings/modelSettings.styles.ts

```diff
@@ -0,0 +1,102 @@
+/**
+ *  Copyright (C) 2017 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import { COLOR } from '../../styles/colors';
+import { Form } from 'formik';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import TextField from '@material-ui/core/TextField';
+import FormControl from '@material-ui/core/FormControl';
+import Chip from '@material-ui/core/Chip';
+import Icon from '@material-ui/core/Icon';
+
+export const StyledTextField = styled(TextField)``;
+
+export const FieldsRow = styled(Grid)`
+	${StyledTextField} {
+		width: 100%;
+		margin-left: 12px;
+		margin-right: 12px;
+	}
+
+	${StyledTextField}:nth-child(2n) {
+		margin-left: 12px;
+	}
+
+	${StyledTextField}:nth-child(2n + 1) {
+		margin-left: 0;
+	}
+
+	${StyledTextField}:last-child {
+		margin-right: 0;
+	}
+`;
+
+export const SelectWrapper = styled(FormControl)`
+	&& {
+		margin-left: 12px;
+		margin-top: 15px;
+	}
+`;
+
+export const TopicTypesContainer = styled.div`
+	margin: 30px 0 15px;
+`;
+
+export const StyledChip = styled(Chip)`
```

Comment:
Later if we are going to have another component that uses chips this should be in a more general styles file , not particular for this component. Preferably instead of a styled component, in the theme. Every chip in the app should look the same.